tests/test_iqm_provider.py

```diff
@@ -113,6 +125,9 @@ def test_facade_backend_raises_error_on_remote_execution_fail(adonis_architectur
     when(IQMClient).submit_run_request(...).thenReturn(uuid.uuid4())
     when(IQMClient).get_run(ANY(uuid.UUID)).thenReturn(RunResult.from_dict(result))
     when(IQMClient).get_run_status(ANY(uuid.UUID)).thenReturn(RunStatus.from_dict(result_status))
+    when(requests).get('http://some_url/info/client-libraries', headers=matchers.ANY, timeout=matchers.ANY).thenReturn(
+        get_mock_ok_response({'qiskit-iqm': '1.2.3'})
```

Comment:
a valid response that iqm-client expects is
```
{
  'iqm-client': {
      'min': '10.0',
      'max':'30.0',
  }
}
```

your response works too because iqm-client ignores parsing errors, but it's kind of misleading